cedar-policy-cli/src/lib.rs

```diff
@@ -401,8 +401,12 @@ pub struct FormatArgs {
     pub indent_width: isize,
 
     /// Automatically write back the formatted policies to the input file.
-    #[arg(short, long)]
+    #[arg(short, long, group = "action")]
```

Comment:
Is this group name exposed in the `--help` output? If it is, we probably shouldn't use `action` since `action` has special meaning in Cedar.

Reply:
It's not exposed in `-h`, and it's not exposed when the rule triggers, i.e:
```sh
cargo r format -p cedar-policy-cli/sample-data/tiny_sandboxes/format/formatted.cedar -w -c
   Compiling cedar-policy-cli v4.0.0 (/home/jun/source/forks/cedar/cedar-policy-cli)
    Finished dev [unoptimized + debuginfo] target(s) in 3.52s
     Running `target/debug/cedar format -p cedar-policy-cli/sample-data/tiny_sandboxes/format/formatted.cedar -w -c`
error: the argument '--write' cannot be used with '--check'

Usage: cedar format --policies <FILE> --write

For more information, try '--help'.
```

As far as I can tell, the argument group name is completely internal to clap (their examples have names like `req_flags` which don't make sense to an end user)